tests/test-extensions/vegan-extension/src/main/java/org/mule/test/vegan/extension/VeganFidelityOperation.java

```diff
@@ -19,4 +22,8 @@ public String tryEat(@XmlHints(allowReferences = false) Object food) {
     throw new IllegalArgumentException("I SHALL NEVER EAT " + food.toString());
   }
 
+  public String tryToEatThisListOfMaps(List<Map<String, String>> food) {
```

Comment:
moving forward we will no longer generate XML support for complex parameters in operations but rather rely on DW for that. I wouldn't stress too much about this